---
category: Biomedicine
topic: Clinical Methods
title: Identification of Bacterial Isolate and Antibiotic Profiling
---
A case study concerning a 56 year-old male, was created for the identification of bacterial isolates, as well as, antibiotic profiling. 8 days after admission, deterioration in the patient's condition was seen, displays of pyrexia followed by rigor and hypotension. Bacterial growth from the patient's blood sample was flagged by an automated blood culture system, instigating the requirement for further microbial tests to establish suitability of treatment options (Buchan et al, 2013). Firstly, applications of phenotypic and biochemical identification techniques were carried out, followed by methods of molecular identification to isolate the species of the bacterium. Subsequently, antibiotic profiling was performed in order to confirm effective forms of treatment.

# Methods

Initial testing of the blood culture begins with diagnostic methods originating from the 19th century: Gram staining, a fundamental procedure that allows differentiation between bacterial cells based on the thickness of the peptidoglycan layer (Beveridge et al, 2001). Achieved through comparing the stain response through observable differences in retention. Then, two sub-culturing agar plates containung selective media were used. Baird Parker agar was used as a diagnostic medium due to its proficiency in growth selectivity for *Staphylococci*, a common pathogen. To detemine the staphylococci species, DNase agar was used, which isolates bacterium based on deoxyribonuclease activity (Niskanen et al, 1978; Chen et al, 2015; Zierdt et al, 1970).

Then, using a Catalase test to determine the presence of bacteria producing catalase enzymes, which, if positive, will observably involve the formation of bubbles due to the decomposition of aerobically produced hydrogen peroxide (H2O2) into molecular oxygen and water (Williams et al, 1928). Furthermore, to differentiate between species, a Coagulase test was preformed: visible latex agglutination by coagulase-positive S. aures on a MASTASTAPH test card, which is caused by the presence of enzymatic bacterial-bound coagulase causing the clumping of cell walls together (Chamberlain et al, 2009). Our final biochemical test was performed with the addition of the blood sample to the Microbact 12S Staphylococcal identification System, where identifying the bacterial isolate at a species level would be possible (Ndahi et al, 2013).

The next test series involved molecular identification. The molecular structure of the bacterial isolate, the 0.8 kb fragments (16s rRNA) extracted from the isolate, via boiling and centrifugal methods of DNA extraction, faced amplification through the PCR reaction, together with PerfectTaq Plus MasterMix, primers 8FPL and 806R. PCR results were then verified by electrophoresis, providing results to use in the identification of the origin of sequence by using the nucleotide Basic Local Alignment Search Tool (BLAST). The electrophoresis process works by separating the DNA by their molecular size, where a uniform mass/charge ratio can be formed when DNA like the 0.8 kb PCRamplified DNA fragments are placed in a correct gel matrix and electric field allowing the DNA fragments to migrate along the ladder based on their size, conformation and negatively-charged phosphate backbone attracted to the positively-charged anode (Lee et al, 2012).

In accordance with the pre-existing model results, and with some of my results, the rest provided in the appendixes, the phenotypic morphology of the colonies formed on the blood agar, were found as clusters of cocci and appeared non-elevated, round-shaped and white coloured. The results from selective plating showed growth formation on Baird-Parker agar, indicative of Staphylococci species, however, a lack of DNase activity was seen on the DNA agar (Chen et al, 2015). Furthermore, after Gram staining, assessment of the bacterial isolate under light microscopy returned positive retention of the purple dye in the bacterial cell walls, which is characteristic of Gram-positive bacteria, affirming further possibility of Staphylococci presence (Beveridge et al, 2001; Goering et al, 2013).

To signify the presence of Staphylococci over Streptococci, a catalase test was performed, and demonstrations of catalase activity were found, allowing positive differentiation between the two species, with the bacterial isolate being Staphylococci (Torok et al, 2005). To produce greater specificity in the identification process, a Coagulase test was completed, producing a result that did not confer with the presence of the highly dangerous and virulent Staphylococcus aures (Tong et al, 2015). Instead the results of the Coagulase test indicated the presence of a coagulase-negative Staphylococci isolate.

Then, in accordance with ‘Table 1.1’, so that identification of the organism at a species level could be made, using the results from the Microbact 12S Staphylococcal Identification System, which distinguishes the Staphylococci by causing pH-induced colour change due to their unique utilisation of sugar, as well as, differences between the enzymatic colorimetric detection of produced species-specific indicators (Oxoid, 2016). From the obtained Microbact 12S code, a query ran in the database retrieved a 99.8% expectation that the species of the bacterial isolate from the patient sample was indeed Staphylococcus epidermidis.

Further molecular identification, methods of electrophoresis reveal the DNA fragments were the same length, due to the DNA bands for S. epidermidis positive control being at the same locus as the tested sample. Using BLAST to determine the species based on the obtained nucleotide sequence, the search returned 102 BLAST hits on the query sequence, with S. epidermidis shown in 60 and with the rest representing other species of Staphylococci, bacterium and an avian species. The returned E value for each BLAST hit was 0, indicating that all the hit sequences perfectly matched the query sequence in full.

To establish the potential of antibiotic treatment options, antibiotic profiling was performed, with results shown in ‘Table 1.2’. Due to the lack and no apparent development of inhibition zones, we can state resistance to Tobramycin, Teicoplanin, Linezolid and Vancomycin in S. epidermidis. Effective treatment choices, where antibiotic sensitivity was present, were found with the use of Gentamicin and Ciprofloxacin. S. epidermidis demonstrated a resistance against all Gram-negative treatment options.

Tobramycin is an aminoglycoside antibiotic which we show S. epidermidis to have resistance to, and according to study, this is most likely due to the presence of bacterial derived enzymes, carefully called Tobramycin Adenylytranferase, capable of adenylylation of Tobramycin, effectively inactivating the antibiotic (Santanam et al, 1976). Use of Teicoplanin showed resistance, explanation of this can be found in the presence of staphylococci derived extracellular slime, which is purposed for increasing the adhesivity to biomaterial furthering the infection, this extracellular slime can negatively impact the therapeutic effect of the antibiotic (Mathur et al, 2005). Although studies do show damage to the presence of S. epidermis when using Teicoplanin, however the seen reactive increase in biofilm production demonstrates the inefficacy of the antibiotic (Claessens et al, 2014).

S. epidermidis was resistant to Linezolid, with further studies confirming recoveries of other resistant isolates (Barros et al, 2014). This resistance may occur due to mutations in the 23S rRNA binding site, ribosomal proteins of the ribosome’s peptide translocation centre, as well as, procurement of ribosomal methyltransferase gene, cfr (Gu et al, 2013). Further antibiotic profiling showed S. epidermidis resistant to Vancomycin, however, this is complicated as some studies demonstrate sensitivity to the antibiotic, with susceptibility increased at greater antibiotic concentrations (Pinheiro et al, 2016). On the other hand, Vancomycin resistance can be seen due to the formation of biofilm, thickness of the cell wall and atlE gene expression (Hellmark et al, 2009; Gazzola et al, 2008).

Nonetheless, in response to our antibiotic profiling tests, sensitivity was demonstrated with the use of both Gentamicin and Ciprofloxacin. Gentamicin has already exemplified an effectivity in acne patients where there are harbours of Staphylococci species on the skin (Khan et al, 2015). The antibiotic functions through blocking the ribosomal units, such as: bacterial specific 30S unit preventing protein translation, as well as, binding with the bacterial rRNA on the 16S subunit which negatively impacts S. epidermidis’s ability in protein-proofreading resulting in the accumulation of mistranslated proteins and decreased cellular functionality (Ammann et al, 2018; Jones et al, 2015).

Furthermore, Ciprofloxacin prevents the progression and limits the density of biofilm through inhibition of DNA gyrase activity and DNA synthesis (Szczuka et al, 2017). Ciprofloxacin also inhibits S. epidermidis adhesion to vascular catheters reducing infection, however, developing resistances can be rapid, and this is a possibility for most antibiotics due to various plasmid exchanges that can result in the genetic transference of resistance (Burnie et al, 1997; Davison, 1999).

# Conclusion

To conclude, we can identify the source of infection to be S. Epidermidis. Effective choice of antibiotic treatment would be Ciprofloxacin or Gentamicin.

# References

Ammann, C. G., Neuhauser, D., Eberl, C., Nogler, M. and Coraca-Huber, D. (2018) Tolerance towards gentamicin is a function of nutrient concentration in biofilms of patient-isolated Staphylococcus epidermidis. Folia Microbiologica. 63 (3) pp. 299-305. DOI: 10.1007/s12223-017-0568-x

Barros, M., Branquinho, R., Grosso, F., Peixe, L. and Novais, C. (2014) Linezolid-Resistant Staphylococcus epidermidis, Portugal, 2012. Emerging Infectious Diseases. 20 (5) pp. 903-905. DOI: 10.3201/eid2005.130783

Beveridge, T. J. (2001) Biotechnic & Histochemistry: official publication of the Biological Stain Commission. 76(3) pp. 111-118

Buchan, W. B., Ginocchio, C. C., Manii, R., Cavagnolo, R., Pancholi, P., Swyers, L., Thomson, R. B., Anderson, C., Kaul, K., and Ledeboer, N. A. (2013) Multiplex Identification of Gram-Positive Bacteria and Resistance Determinants Directly from Positive Blood Culture Broths: Evaluation of an Automated Microarray-Based Nucleic Acid Test. PLOS Medicine. 10 (7) DOI: 10.1371/journal.pmed.1001478

Burnie, J. P. and Loudon, K. W. (1997) Ciprofloxacin-resistant Staphylococcus epidermidis and hands. Correspondence. 349 (9052) pp. 649. DOI: 10.1016/S0140-6736(05)61593-0

Chamberlain, N. (2009) Coagulase test for staphylococcus species. American Society for Microbiology Microbe Library. [Accessed on 2019 October 26]. Available from: https://www.asmscience.org/content/education/imagegallery/image.3207

Chen, J., Tang, J., Bhunia, A. K., Tang, C., Wang., and Shi, H. (2015) Development of a multi-pathogen enrichment broth for simultaneous growth of five common foodborne pathogens. The Journal of general and applied microbiology. 61(6) pp. 224-231. DOI: 10.2323/jgam.61.224.

Claessens, J., Roriz, M., Baatsen, P., Van Mellaert, L. and Van Eldere, J. (2014) Inefficacy of vancomycin and teicoplanin in eradicating and killing Staphylococcus epidermis biofilms in vitro, International Journal of Antimicrobial Agents. DOI: 10.1016/j.ijantimicag.2014.11.011

Davison, J. (1999) Genetic exchange between bacteria in the environment. Plasmid. 42 (2) pp. 73-91. DOI: 10.1006/plas.1999.1421

Gazzola, S. and Cocconcelli, P. S. (2008) Vancomycin heteroresistance and biofilm formation in Staphylococcus epidermidis from food. Microbiology. 154 pp. 3224-3231. DOI: 10.1099/mic.0.2008/021154-0

Goering, R. V., Dockrell, H. M., Zuckerman, M., Roitt, I. M. & Chiodini, P. L. (2013) Mims’ medical microbiology. (5th ed.) Philadelphia: Mosby Elsevier. pp. 7-8

Gu, B., Kelesidis, T., Tsiodras, S., Hindler, J. and Humphries, R. M. (2013) The emerging problem of linezolid-resistant Staphylococcus. Journal of Antimicrobial Chemotherapy. 68 (1) pp. 4-11. DOI: 10.1093/jac/dks354

Hellmark, B., Unemo, M., Nilsdotter-Augustinsson, A. and Soderquist, B. (2009) Antibiotic susceptibility among Staphylococcus epidermidis isolated from prosthetic joint infections with special focus on rifampicin and variability of rpoB gene. Clinical Microbiology and Infection. 15 (3) pp. 238-244. DOI: 10.1111/j.1469-0691.2008.02663.x

Jones, P. V., Huey, S., Davis, P., McLemore, R., McLaren, A. and Hayes, M. A. (2015) Biophysical separation of Staphylococcus epidermidis strains based on antibiotic resistance. The Analyst. 140 (15) pp. 5152-5161. DOI: 10.1039/c5an00906e

Khan, A. F., Hana, H. K., Sheak, J. and Begum, K. (2015) Antibiotic Sensitivity of Staphylococcus aureus and Staphylococcus epidermidis Isolated from Acne Patients. Bangladesh Pharmaceutical Journal. 18 (2) pp. 121-125.

Lee, P. Y., Costumbrado, J., Hsu, C., and Kim, Y. H. (2012) Agarose Gel Electrophoresis for the Separation of DNA Fragments, Journal of Visualized Experiments. 62 (3923). DOI: 10.3791/3923

Mathur, T., Singhal, S., Khan, S., Uadhyay, D., Fatma, T. and Rattan, A. (2005) Adverse effect of staphylococci slime on in vitro activity of glycopeptides, Japanese Journal of Infectious disease. 58 (6) pp. 353-357.

Ndahi, M. D., Kwaga, J. K. P., Bello, M., Kabir, J., Umoh, V. J., Yakubu, S. E., and, Nok, A. J. (2013) Prevalence and antimicrobial susceptibility of Listeria monocytogenes and methicillin‐resistant Staphylococcus aureus strains from raw meat and meat products in Zaria, Nigeria, Letters in Applied Microbiology. 58 (3). DOI: 10.1111/lam.12183

Niskanen, A., Aalto, M. (1978) Comparison of selective media for coagulase-positive enterotoxigenic Staphylococcus aureus. Applied and Environmental Microbiology. 135(6) pp.1233–1236.

Oxoid (2016) Microbact Biochemical Identification Kits; Information from Manufacturer. Available at: http://www.oxoid.com/UK/blue/prod_detail/prod_detail.asp?pr=MB1561 (Accessed on 06/12/2019)

Pinheiro, L., Brito, C. I., Pereira, V. C., Oliveira, A., Bartolomeu, A. R., Camargo, C. H. and Cunha, M. L. R. S. (2016) Susceptibility Profile of Staphylococcus epidermidis and Staphylococcus haemolyticus Isolated from Blood Cultures to Vancomycin and Novel Antimicrobial Drugs over a Period of 12 Years. Microbial Drug Resistance. 22 (4). DOI: 10.1089/mdr.2015.0064

Santanam, P. and Kayser, F. H. (1976) Tobramycin adenylyltransferase: a new aminoglycosideinactivating enzyme from Staphylococcus epidermidis, The Journal of Infectious Diseases. 134: (Supplement) pp. 33-39. DOI: 10.1093/infdis/134.supplement_1.s33

Szczuka, E., Jablonska, J. and Kaznowski, A. (2017) Effect of subinhibitory concentrations of tigecycline and ciprofloxacin on the expression of biofilm-associated genes and biofilm structure of Staphylococcus epidermidis. Microbiology. 163 (5). DOI: 10.1099/mic.0.000453

Torok, E. and Day, N. (2005) Staphylococcal and streptococcal infections, Medicine. 33 (5) pp. 97-100. DOI: 10.1383/medc.33.5.97.64964

Tong, S. Y. C., Davis, J. S., Eichenberger, E., Holland, T. L., and Fowler, Jr, V. G. (2015) Staphylococcus aureus Infections: Epidemiology, Pathophysiology, Clinical Manifestations, and Management, Clinical Microbiology Reviews. 28 (3) pp. 603-661. DOI: 10.1128/CMR.00134-14

Williams, J. (1928) The Decomposition of Hydrogen Peroxide By Liver Catalase. Journal of General Physiology. 11 (4) pp. 309. DOI: 10.1085/jgp.11.4.309

Zierdt, C. H., and Golde, D. W. (1970) Deoxyribonuclease-Positive Staphylococcus epidermidis Strains. Applied Microbiology. 20(1) pp. 54-57

Sequence used in BLAST
AGAAAGGAGGTGATCCAGCCGCACCTTCCGATACGGCTACCTTGTTACGACTTCACCCCA
ATCATTTGTCCCACCTTCGACGGCTAGCTCCAAATGGTTACTCCACCGGCTTCGGGTGTTA
CAAACTCTCGTGGTGTGACGGGCGGTGTGTACAAGACCCGGGAACGTATTCACCGTAGC
ATGCTGATCTACGATTACTAGCGATTCCAGCTTCATATAGTCGAGTTGCAGACTACAATC
CGAACTGAGAACAACTTTATGGGATTTGCTTGACCTCGCGGTTTCGCTACCCTTTGTATTG
TCCATTGTAGCACGTGTGTAGCCCAAATCATAAGGGGCATGATGATTTGACGTCATCCCC
ACCTTCCTCCGGTTTGTCACCGGCAGTCAACTTAGAGTGCCCAACTTAATGATGGCAACT
AAGCTTAAGGGTTGCGCTCGTTGCGGGACTTAACCCAACATCTCACGACACGAGCTGAC
GACAACCATGCACCACCTGTCACTCTGTCCCCCGAAGGGGAAAACTCTATCTCTAGAGGG
GTCAGAGGATGTCAAGATTTGGTAAGGTTCTTCGCGTTGCTTCGAATTAAACCACATGCT
CCACCGCTTGTGCGGGTCCCCGTCAATTCCTTTGAGTTTCAACCTTGCGGTCGTACTCCCC
AGGCGGAGTGCTTAATGCGTTAGCTGCAGCACTAAGGGGCGGAAACCCCCTAACACTTA
GCACTCATCGTTTACGGCGTGGACTACCAGGGTATCTAATCCTGTTTGATCCCCACGCTTT
CGCACATCAGCGTCAGTTACAGACCAG AA
